.github/workflows/coverage.yml

```
@@ -0,0 +1,60 @@
+name: Generate code coverage badge
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Update coverage badge
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Should use `actions/checkout@v3` from this time period

```suggestion
        uses: actions/checkout@v3
```